client/main.lua

```diff
@@ -81,86 +81,69 @@ local function CreateZone(type, number)
-        label = Lang:t("label.payslip")
+        label = locale("label.payslip")
         coords = sharedConfig.locations["main"].coords.xyz
         heading = sharedConfig.locations["main"].coords.w
         boxName = sharedConfig.locations["main"].label
-        size = 3
+        size = vec3(3,3,10)
     elseif type == "vehicle" then
         event = "qb-tow:client:Vehicle"
-        label = Lang:t("label.vehicle")
+        label = locale("label.vehicle")
         coords = sharedConfig.locations["vehicle"].coords.xyz
         heading = sharedConfig.locations["vehicle"].coords.w
         boxName = sharedConfig.locations["vehicle"].label
-        size = 5
+        size = vec3(5,5,10)
     elseif type == "towspots" then
         event = "qb-tow:client:SpawnNPCVehicle"
-        label = Lang:t("label.npcz")
+        label = locale("label.npcz")
         coords = sharedConfig.locations[type][number].coords.xyz
         heading = sharedConfig.locations["towspots"][number].coords.w --[[@as number?]]
         boxName = sharedConfig.locations["towspots"][number].name
-        size = 50
+        size = vec3(50,50,10)
     end
 
     if config.useTarget and type == "main" then
-        exports['qb-target']:AddBoxZone(boxName, coords, size, size, {
-            minZ = coords.z - 5.0,
-            maxZ = coords.z + 5.0,
+        exports.ox_target:addBoxZone({
             name = boxName,
-            heading = heading,
-            debugPoly = false,
-        }, {
+            coords = coords,
+            size = size,
+            rotation = heading,
+            debug = config.debugPoly,
             options = {
                 {
-                    type = "client",
                     event = event,
                     label = label,
-                },
-            },
-            distance = 2
+                    distance = 2,
+                }
+            }
         })
     else
-        local zone = BoxZone:Create(
-            coords, size, size, {
-                minZ = coords.z - 5.0,
-                maxZ = coords.z + 5.0,
-                name = boxName,
-                debugPoly = false,
-                heading = heading,
-            })
-
-        local zoneCombo = ComboZone:Create({zone}, {name = boxName, debugPoly = false})
-        zoneCombo:onPlayerInOut(function(isPointInside)
-            if isPointInside then
-                if type == "main" then
-                    TriggerEvent('qb-tow:client:PaySlip')
-                elseif type == "vehicle" then
-                    TriggerEvent('qb-tow:client:Vehicle')
-                elseif type == "towspots" then
-                    TriggerEvent('qb-tow:client:SpawnNPCVehicle')
-                end
-            end
-        end)
+        local zone = lib.zones.box({
+            coords = coords,
+            size = size,
+            rotation = heading,
+            debug = config.debugPoly,
+            event = event,
+            inside = function(self)
```

Comment:
I had to change this to `onEnter` or else the context menu spams

Reply:
good catch